### PR TITLE
[PW-7634] - Generate BinLookup

### PR DESCRIPTION
**Pull request by antolo-arch**
**Description**
Generate BinLookup

**Tested scenarios**
UTs

### CODE REVIEW

**Review by AlexandrosMor — APPROVED** [review]

**Review by candemiralp — APPROVED** [review]

**Review by candemiralp — APPROVED** [review]

File: Adyen/services/binLookup.py [hunk 1]
```diff
@@ -1,24 +1,42 @@
...
     """
 
     def __init__(self, client=None):
-        super(AdyenBinLookup, self).__init__(client=client)
-        self.service = "BinLookup"
+        super(AdyenBinlookupApi, self).__init__(client=client)
+        self.service = "binlookup"
 
-    def get_cost_estimate(self, request="", **kwargs):
+    def check_if3d_secure_is_available(self, request, idempotency_key=None, **kwargs):
```
Comment (michaelpaul) [inline Adyen/services/binLookup.py:25]:
```suggestion
    def get_3ds_availability(self, request, idempotency_key=None, **kwargs):
```

File: Adyen/services/binLookup.py [hunk 2]
```diff
@@ -1,24 +1,42 @@
...
+        """
+        Check if 3D Secure is available
+        """
+        endpoint = f"/get3dsAvailability"
+        endpoint = endpoint.replace('/', '', 1)
+        method = "POST"
+        return self.client.call_adyen_api(request, self.service, method, endpoint, idempotency_key, **kwargs)
 
-        action = "getCostEstimate"
+    def get_fees_cost_estimate(self, request, idempotency_key=None, **kwargs):
```
Comment (michaelpaul) [inline Adyen/services/binLookup.py:34]:
```suggestion
    def get_cost_estimate(self, request, idempotency_key=None, **kwargs):
```

**Review by michaelpaul — CHANGES_REQUESTED** [review]
Let's use manual names for this service since it's so small.

File: Adyen/services/binLookup.py [hunk 3]
```diff
@@ -1,24 +1,42 @@
...
...` header with the API key as the value, for example:   ``` curl -H \"Content-Type: application/json\" \\ -H \"X-API-Key: YOUR_API_KEY\" \\ ... ```  Alternatively, you can use the username and password to connect to the API using basic authentication, for example:  ``` curl -U \"ws@Company.YOUR_COMPANY_ACCOUNT\":\"YOUR_BASIC_AUTHENTICATION_PASSWORD\" \\ -H \"Content-Type: application/json\" \\ ... ```  ## Versioning The BinLookup API supports [versioning](https://docs.adyen.com/development-resources/versioning) using a version suffix in the endpoint URL. This suffix has the following format: \"vXX\", where XX is the version number.  For example: ``` https://pal-test.adyen.com/pal/servlet/BinLookup/v52/get3dsAvailability ```## Going live  To authneticate to the live endpoints, you need an [API credential](https://docs.adyen.com/development-resources/api-credentials) from your live Customer Area.  The live endpoint URLs contain a prefix which is unique to your company account: ```  https://{PREFIX}-pal-live.adyenpayments.com/pal/servlet/BinLookup/v52/get3dsAvailability ```  Get your `{PREFIX}` from your live Customer Area under **Developers** > **API URLs** > **Prefix**.  # noqa: E501
```
Comment (michaelpaul) [inline Adyen/services/binLookup.py:4]: We can omit these long descriptions from code. Its meant for API explorer where it renders properly.

Comment (antolo-arch) [reply Adyen/services/binLookup.py:4]: should we remove the partial header from the template then?

**Review by wboereboom — APPROVED** [review]

**Review by jillingk — APPROVED** [review]

**Review by candemiralp — APPROVED** [review]

**Pull request merged**